examples/contrib/epidemiology/sir.py

```diff
@@ -92,16 +93,17 @@ def hook_fn(kernel, *unused):
                      warmup_steps=args.warmup_steps,
                      num_samples=args.num_samples,
                      num_chains=args.num_chains,
+                     mp_context="spawn" if parallel else None,
```

Comment:
unfortunately i've found this to be somewhat buggy (fair number of crashes)

Reply:
do you use "forkserver" instead?

@neerajprad do you have suggestions?

Reply:
no i've used spawn. but i've tried to avoid it...